### PR TITLE
add DB_USERNAME & DB_PASSWORD to lambo

### PR DESCRIPTION
**Pull request by Naoray**
For me it would be helpful to be able to specify username and password for database in the lambo config.

What do you think?

### CODE REVIEW

File: lambo [hunk 1]
```diff
@@ -175,6 +179,8 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
+DB_USERNAME="root"
+DB_PASSWORD="secret"
```
Comment (mattstauffer) [inline lambo]: The default will need to be an empty string.

File: lambo [hunk 2]
```diff
@@ -319,6 +333,14 @@ if [[ "$AUTH" = true ]]; then
     php artisan make:auth
 fi
 
+if [[ "$DB_USERNAME" != "" ]]; then
```
Comment (mattstauffer) [inline lambo]: What are these lines for?

Comment (Naoray) [reply lambo]: TBH ... not sure why I put them there. :see_no_evil:

File: lambo (hunk 1 above)
Comment (Naoray) [reply lambo]: changed

**Comment by Naoray** [comment]
Fine now?

File: lambo [hunk 3]
```diff
@@ -175,6 +179,8 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
+DB_USERNAME=""
```
[the remark this answers is not in the document]
Comment (Naoray) [reply lambo]: Changed

Comment (mattstauffer) [inline lambo]: Please make it root here, as well, in case someone doesn't have a config file. Thanks!

Comment (Naoray) [reply lambo]: Sorry :see_no_evil:, should have changed now

**Comment by Naoray** [comment]
Guess I resolved every change request @mattstauffer

**Comment by mattstauffer** [comment]
Thanks @Naoray! We're merged into `dev-master` for folks to test and I'll make a release later.

**Pull request merged**